### PR TITLE
feat: add support for webContents.executeJavaScript

### PR DESCRIPTION
**Pull request by magne4000**
Fixes https://github.com/electron/spectron/issues/163

### CODE REVIEW

**Comment by codebytere** [comment]
@magne4000 would you mind rebasing this? that should fix spec failures i believe.

File: lib/api.js [hunk 1]
```diff
@@ -385,6 +387,22 @@ Api.prototype.addSavePageSupport = function () {
   }
 }
 
+Api.prototype.addExecuteJavaScriptSupport = function () {
+  var app = this.app
```
Comment (codebytere) [inline lib/api.js]: these should all be `const` variables

File: lib/api.js [hunk 2]
```diff
@@ -385,6 +387,22 @@ Api.prototype.addSavePageSupport = function () {
...
+  var self = this
+
+  app.client.addCommand('webContents.executeJavaScript', function (code, useGesture) {
+    return this.executeAsync(function (code, useGesture, requireName, done) {
+      var webContents = window[requireName]('electron').remote.getCurrentWebContents()
```
Comment (codebytere) [inline lib/api.js]: same as above

**Comment by magne4000** [comment]
@codebytere it's all good 👍

**Pull request merged**